airflow/jobs/scheduler_job_runner.py

```diff
@@ -1427,6 +1463,53 @@ def _send_sla_callbacks_to_processor(self, dag: DAG) -> None:
         )
         self.job.executor.send_callback(request)
 
+    @provide_session
+    def _fail_tasks_stuck_in_queued(self, session: Session = NEW_SESSION) -> None:
+        """
+        Mark tasks stuck in queued for longer than `task_queued_timeout` as failed.
+
+        If one of the configuration settings `kubernetes.worker_pods_pending_timeout`,
+        `celery.stalled_task_timeout`, or `celery.task_adoption_timeout` is set and
+        has a higher value than `scheduler.task_queued_timeout`, that value will be
+        used as `task_queued_timeout`.
+
+        Tasks can get stuck in queued for a wide variety of reasons (e.g. celery loses
+        track of a task, a cluster can't further scale up its workers, etc.), but tasks
+        should not be stuck in queued for a long time. This will mark tasks stuck in
+        queued for longer than `self._task_queued_timeout` as failed. If the task has
+        available retries, it will be retried.
+        """
+        self.log.debug("Calling SchedulerJob._fail_tasks_stuck_in_queued method")
+        for attempt in run_with_db_retries(logger=self.log):
+            with attempt:
+                self.log.debug(
+                    "Running SchedulerJob._fail_tasks_stuck_in_queued with retries. Try %d of %d",
+                    attempt.retry_state.attempt_number,
+                    MAX_DB_RETRIES,
+                )
+                try:
+                    query = session.query(TI).filter(
+                        TI.state == State.QUEUED,
+                        TI.queued_dttm < (timezone.utcnow() - timedelta(seconds=self._task_queued_timeout)),
+                    )
+                    tasks_stuck_in_queued: list[TaskInstance] = with_row_locks(
+                        query, of=TI, session=session, **skip_locked(session=session)
+                    ).all()
+                    print(f"\n\ntis: {tasks_stuck_in_queued}\n\n")
```

Comment:
```suggestion
                    print(f"\n\ntis: {tasks_stuck_in_queued}\n\n")
```

Leftover debugging? Or, switch this to debug or warning log instead (with some extra context).